src/plugins/data_view_editor/public/components/empty_prompts/empty_prompts.tsx

```diff
@@ -94,11 +95,16 @@ export const EmptyPrompts: FC<Props> = ({ allSources, onCancel, children, loadSo
       // first time
       return (
         <>
-          <EmptyIndexPatternPrompt
-            goToCreate={() => setGoToForm(true)}
-            indexPatternsIntroUrl={docLinks.links.indexPatterns.introduction}
-            canSaveIndexPattern={dataViews.getCanSaveSync()}
-          />
+          <EuiFlexGroup justifyContent={'spaceAround'} direction={'column'}>
+            <EuiFlexItem grow={false}>
+              <NoDataViewsComponent
+                onClickCreate={() => setGoToForm(true)}
+                canCreateNewDataView={application.capabilities.indexPatterns.save as boolean}
```

Comment:
The `as` shouldn't be necessary, but you can always do a `!!application.capabilities.indexPatterns.save`
<img width="419" alt="Screen Shot 2022-03-19 at 11 19 21 PM" src="https://user-images.githubusercontent.com/297604/159147970-f532f66f-68f1-4409-aff3-f0020adcce51.png">